src/aap_eda/services/activation/manager.py

```diff
@@ -1018,12 +1019,14 @@ def update_logs(self):
             return
 
     def _create_activation_instance(self):
+        fqcn = get_fully_qualified_name(self.db_instance)
```

Comment:
@Alex-Izquierdo We also need a get_parent method that does something like this

```
from django.core.exceptions import ObjectDoesNotExist
from django.utils.module_loading import import_string
def get_parent(self):
      try:
             klass = import_string(self.parent_fqcn)
            return klass.objects.get(id=self.parent_id)
    except ObjectDoesNotExist:
        LOGGER.exception(
            f"Parent id  {self.parent_id} for class {self.parent_fqcn} no longer exists, "
            )
 ````